lua/treesitter-context/render.lua

```diff
@@ -151,7 +151,7 @@ local function highlight_contexts(bufnr, ctx_bufnr, contexts)
           local msrow = offset + (nsrow - start_row)
           local merow = offset + (nerow - start_row)
 
-          local hl = buf_query.hl_cache[capture]
+          local hl = buf_query:get_hl_from_capture(capture)
```

Comment:
I don't get why the tests are failing, i only changed this in highlight_contexts.